app/src/main/java/com/example/moviedb/data/repository/MovieRepository.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.moviedb.data.repository
+
+import com.example.moviedb.data.service.MoviesResponse
+import com.example.moviedb.data.service.api.ApiService
+import io.reactivex.Single
+
+class MovieRepository(
+    val service: ApiService
```

Comment:
thuộc tính này không nhất thiết phải để public chứ nhỉ? 
```suggestion
    private val service: ApiService
```